### PR TITLE
Load Hotel Restaurant WebClip content from file

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1392
[Problem] HTML content is loaded from JS file
[Solution] Use fetch API in order to load html file content

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
After making this PR I have some doubts if we really need to have custom element in this case.
This is particular instance of webclip and app-hotel-breakfast will always exist only once.

File: examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js [hunk 1]
```diff
@@ -1,107 +1,6 @@
 /* global tau */
-const tmpl = document.createElement("template");
```
Comment (DohyungLim) [inline examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:2]: Is there any reason to move it to html?
Is it necessary to switch pages in order for webclip to work?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js (hunk 1 above)
Comment (singa2000) [reply examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:2]: I request Lukasz to move it to html.
We should be able to use it in the same form as general templates.

File: examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                        <div class="ui-li-text">
+                            <span class="ui-li-text-title">
+                                English Breakfast
+                            </span>
+                            <span class="ui-li-text-sub">
+                                Delicious breakfast: fried egg, baked bacon, beans, sausages, tomato, bread, hot drink.
+                            </span>
+                        </div>
+                        <div class="app-hotel-order">
+                            <a href="breakfast-menu.html" class="ui-btn ui-btn-contained-colored" data-inline="true">Order</a>
```
Comment (singa2000) [inline examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast.html:17]: The "breakfast-menu.html" and "payment.html" file also move to /HomeApp/webclip/hotel-breakfast.html , please.

**Comment by singa2000** [comment]
I also  doubts if we really need to have custom element in this case.
Why we need the "hotel-breakfast.html" file?
Why we don't put the code into "webclip.html" file?

**Comment by lslachciak** [comment]
I prepared new PR which resigns from custom element. 
https://github.com/Samsung/TAU/pull/1394
Also this new PR will fix issue when readding HotelRestaurant WebClip

**Pull request closed without merging**